src/Commands/CreateTables.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace ActiveCollab\JobQueue\Commands;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Exception;
+
+/**
+ * @package ActiveCollab\JobQueue\Commands
+ */
+class CreateTables extends Command
+{
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('create_tables')->setDescription('Create tables that are needed for MySQL queue to work');
+    }
+
+    /**
+     * @param  InputInterface $input
+     * @param  OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $connection = $this->getDatabaseConnection($input);
+
+            $connection->execute("CREATE TABLE IF NOT EXISTS  `jobs_queue` (
```

Comment:
Check if these tables definitions are up to date. Consider adding `createTables` method to `MySqlQueue`